pkg/apis/pgv2.percona.com/v2/perconapgcluster_types.go

```diff
@@ -47,6 +47,10 @@ type PerconaPGCluster struct {
 }
 
 type PerconaPGClusterSpec struct {
+
+	// +optional
+	Metadata *Metadata `json:"metadata,omitempty"`
```

Comment:
no need to redefine it in this package, you can directly use `Metadata *crunchyv1beta1.Metadata`

Reply:
also I don't see we're passing this value to Crunchy CR anywhere, we need to